packages/text-field/index.tsx

```diff
@@ -394,12 +407,13 @@ class TextField<
 
   renderHelperLine(
     helperText?: React.ReactElement<HelperTextProps>,
-    characterCounter?: React.ReactElement<any>
+    characterCounter?: React.ReactElement<CharacterCounterProps>
   ) {
+    const el = this.textFieldElement.current;
     return (
-      <div className={cssClasses.HELPER_LINE}>
+      <div className={cssClasses.HELPER_LINE} style={{width: el ? `${el.clientWidth}px` : 'auto'}}>
```

Comment:
We shouldn't have `style` here - this should use the MDC Web's styles. Is this for the screenshot tests?

Reply:
I thought there is a spec that helper-line width.
But it looks my misunderstanding...
Thank you!